qcengine/programs/gcp.py

```diff
@@ -268,3 +269,46 @@ def parse_output(self, outfiles: Dict[str, str], input_model: "AtomicInput") ->
 
         output_data["success"] = True
         return AtomicResult(**{**input_model.dict(), **output_data})
+
+
+class MCTCGCPHarness(GCPHarness):
+
+    _defaults = {
+        "name": "MCTC-GCP",
+        "scratch": True,
+        "thread_safe": True,
+        "thread_parallel": False,
+        "node_parallel": False,
+        "managed_memory": False,
+    }
+    version_cache: Dict[str, str] = {}
+
+    @staticmethod
+    def found(raise_error: bool = False) -> bool:
+        return which(
+            "mctc-gcp",
+            return_bool=True,
+            raise_error=raise_error,
+            raise_msg="Please install via `conda install gcp -c psi4`.",
+        )
+
+    def get_version(self) -> str:
+        self.found(raise_error=True)
+
+        version = None
+        which_prog = which("mctc-gcp")
+        if which_prog not in self.version_cache:
+            # option not (yet) available, instead find in help output
```

Comment:
Misleading comment here, this is actually available.
```suggestion
```

Reply:
i've adjusted the version slurping